container/raw/handler.go

```diff
@@ -77,6 +77,9 @@ func newRawContainerHandler(name string, cgroupSubsystems *libcontainer.CgroupSu
 	pid := 0
 	if isRootCgroup(name) {
 		pid = 1
+
+		// delete pids from cgroup paths because /sys/fs/cgroup/pids/pids.current not exist
+		delete(cgroupPaths, "pids")
```

Comment:
This means we won't have current or max threads for the root cgroup, right?

That would be useful to have if we want to have the kubelet perform eviction when we are running low.  I don't think it needs to be a requirement for this PR, but we should consider adding this as a future improvement.  We would want to get the limit from `/proc/sys/kernel/threads-max`, and (probably?) would have to scan `/proc` for total thread count.